scripts/utils/trading_strategy_helpers.js

```diff
@@ -119,10 +77,11 @@ module.exports = function (web3 = web3, artifacts = artifacts) {
 
   const globalTokenPromisesFromAddress = {}
   /**
-   * Queries EVM for ERC20 token details by address
-   * and returns a list of promises of detailed token information.
+   * Queries EVM for ERC20 token details by address and returns a list of promises of detailed token information.
+   *
    * @param {Address[]} tokenAddresses list of *unique* token addresses whose data is to be fetch from the EVM
-   * @return {Promise<TokenObject>[]} list of detailed/relevant token information
+   * @param {boolean} [debug=false] prints log statements when true
+   * @returns {Promise<TokenObject>[]} list of detailed/relevant token information
```

Comment:
This type was actually wrong, since the output is a mapping from addresses to promises of token objects. So, if we want to be formally correct we should use the much less informative `object`.
```suggestion
   * @returns {object} object mapping token addresses to a promise of relevant token information
```